packager/mpd/base/mpd_utils.cc

```diff
@@ -419,6 +441,11 @@ void AddContentProtectionElementsHelperTemplated(
       if (!entry.pssh().empty()) {
         drm_content_protection.subelements.push_back(
             GenerateCencPsshElement(entry.pssh()));
+        if(entry.uuid() == kPlayReadyUUID && protected_content.include_mspr_pro()) {
+          drm_content_protection.subelements.push_back(
+              GenerateMsprProElement(entry.pssh()));
+          drm_content_protection.value = kContentProtectionValueMSPR20;
```

Comment:
Should we set the value to "MSPR 2.0" regardless of "include_mspr_pro"?

Reply:
I don't think it is necessary.
Also some consumers may expect no 'value'...